### ISSUE TITLE
Support for 1.0.4? 

### ISSUE THREAD

**Issue by JeffreyConnected** [wmurphyrd/aframe-super-hands-component#171]
While testing this component, I used v1.0.4 and got this error:

**`core:a-node:error Failure loading node:   TypeError: o is undefined`**

I was testing this basic example:

```
<head>
  <title>Most Basic Super-Hands Example</title>
  <script src="https://aframe.io/releases/0.8.2/aframe.min.js"></script>
  <script src="https://cdn.rawgit.com/donmccurdy/aframe-extras/v4.1.2/dist/aframe-extras.min.js"></script>
  <script src="https://unpkg.com/super-hands@3.0.0/dist/super-hands.min.js"></script>
</head>

<body>
  <a-scene>
    <a-assets></a-assets>
    <a-entity>
      <a-camera></a-camera>
      <a-entity sphere-collider="objects: a-box" super-hands hand-controls="left"></a-entity>
      <a-entity sphere-collider="objects: a-box" super-hands hand-controls="right"></a-entity>
    </a-entity>
    <!-- hover & drag-drop won't have any obvious effect without some additional event handlers or components. See the examples page for more -->
    <a-box hoverable grabbable stretchable draggable dropppable color="blue" position="0 0 -1"></a-box>
  </a-scene>
</body>
```

**Comment by QuinnCiccoretti** [comment]
This is an issue with aframe. See this issue:
https://github.com/aframevr/aframe/issues/4645

**Comment by wmurphyrd** [comment]
v3.0.1 released for compatibility with A-Frame v1.0.4 thanks
